homeassistant/components/airzone/strings.json

```diff
@@ -8,12 +8,19 @@
       "invalid_system_id": "Invalid Airzone System ID"
     },
     "step": {
+      "discovered_connection": {
+        "data": {
+          "id": "System ID",
```

Comment:
You can reference the `id` in the user flow so translators don't have to translate this twice.

There is an example in `homekit/strings.json`

Reply:
Done in 464be71c021926535dacaebdbda7d097441486ed